tests/target/single_line_if/false_50.rs

```diff
@@ -0,0 +1,61 @@
+// rustfmt-single_line_if_else_max_width: 50
```

Comment:
Even though it's `false` could you explicitly add the configuration, Also, did you mean to use  `single_line_if_else_max_width` or your new `single_line_simple_if_max_width`?

If you were explicitly testing compatibility with `single_line_if_else_max_width` I'd prefer that be done in a standalone test file, though I'm leaning towards reusing `single_line_if_else_max_width` as the width configuration.

Reply:
I wanted to use `single_line_if_else_max_width`, yes.